### ISSUE TITLE
Nested blocks renderChildren method does not render

### ISSUE THREAD

**Issue by kerkness** [area17/twill#2235]
## Description

When following the [documentation example for making a nested block editor](https://twillcms.com/docs/block-editor/nested-blocks.html).   The `$renderData->renderChildren('editor_name')` does not render anything.  

### Steps to reproduce

Created block with nested editors
```
@twillBlockTitle('Nested Block left right')
@twillBlockIcon('text')
@twillBlockGroup('app')
 
<x-twill::input
    name="title"
    label="Title"
    :translated="true"
/>
 
<x-twill::block-editor name="left"/> 
<x-twill::block-editor name="right"/> 
```

Created a blade file for rendering
```
<div style="width: 100%">
    <div style="width: 50%; float: left;">
        Left
        {!! $renderData->renderChildren('left') !!}
    </div>
    <div style="width: 50%; float: left;">
        Right
        {!! $renderData->renderChildren('right') !!}
    </div>
</div>
```

Added block to module with content in both left and right nested editors.

I also created a ComponentBlock with the same results.

### Expected result

Blocks render

### Actual result

No errors are thrown and the block blade template is loaded however `$renderData->renderChildren`  method doesn't output anything.   Inspecting `@dump($renderData)` shows that the children and block data is present.   Looking into the `renderChildren` method shows that the parentEditorName is correct however the call to `$child->renderView(...)`  [at line 35 of the RenderData class](https://github.com/area17/twill/blob/3.x/src/Services/Blocks/RenderData.php#L35) is returning an empty string `''`  

### Versions

Twill 3.x  (latest)
Laravel 10.x (latest)

**Comment by kerkness** [comment]
I appear to be having this issue because all of my blocks are  the new Component Blocks and the blade files are not in the standard path.

**Comment by kerkness** [comment]
I guess there are some discrepancies between using a Component Block in a nested block editor vs a primary block.   The `render()` method of a component block doesn't appear to be called when it's used in a nested block editor.   As well you don't have direct access to `$input` or `$translatedInput` methods in the template when used in a nested editor.

The documentation doesn't provide a clear relationship between Component Blocks and Block Classes which is causing me some confusion.

**Comment by kerkness** [comment]
This issue can be closed with [PR #2243](https://github.com/area17/twill/pull/2243)
